core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/MarthaDosResolver.scala

```diff
@@ -12,8 +12,8 @@ import org.broadinstitute.dsde.rawls.util.{HttpClientUtilsStandard, Retry}
 import scala.concurrent.{ExecutionContext, Future}
 
 case class ServiceAccountEmail(client_email: String)
-case class MarthaV2ResponseData(data: Option[ServiceAccountEmail])
-case class MarthaV2Response(googleServiceAccount: Option[MarthaV2ResponseData])
+case class MarthaV2ResponseData(data: ServiceAccountEmail)
+case class MarthaV2Response(googleServiceAccount: MarthaV2ResponseData)
```

Comment:
this will make workflow submission barf, alerting us to badness if for some reason we don't get an SA email back. (which was the case when we were mistakenly not sending a bearer token)

Reply:
do we want this?